docs/docs/supported-providers/supported-providers.md

```diff
@@ -3,7 +3,7 @@
 FlockMTL supports the following providers for semantic analytics tasks such as text generation, embedding creation, and ranking:
 
 - [**Ollama**](/docs/supported-providers/ollama)
-- [**OpenAI**](/docs/supported-providers/openai)
+- [**OpenAI & OpenAI-Compatible Providers**](/docs/supported-providers/openai)
```

Comment:
OpenAI-Compatible Providers* no need for OpenAI & ...

Reply:
done